### PR TITLE
#620 feat(sentry): sentry integration for a client

### PR DESCRIPTION
**Pull request by sergeevNick**
Need to add sentry DSN to envs.

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -51,8 +51,8 @@
     "@ngx-gallery/gallerize": "4.0.3",
     "@ngx-gallery/lightbox": "4.0.3",
     "@node-steam/vdf": "2.1.0",
-    "@sentry/angular": "^7.11.1",
-    "@sentry/tracing": "^7.11.1",
+    "@sentry/angular": "7.11.1",
+    "@sentry/tracing": "7.11.1",
```
Comment (tsa96) [inline client/package.json:55]: This is in a separate commit for some reason, please fixup into the other and force-push (we always make sure to keep the master branch all of Momentum repo's as clean as possible, even if that requires other branches are regularly force-pushed to)

File: client/src/environments/environment.prod.ts [hunk 2]
```diff
@@ -7,4 +7,5 @@ export const environment = {
   production: true,
   api: 'https://api.momentum-mod.org',
   auth: 'https://auth.momentum-mod.org',
+  sentry: { dsn: '', env: 'client-production' },
```
Comment (tsa96) [inline client/src/environments/environment.prod.ts:10]: Even assuming we're happy with the fact we pass the Sentry DSN to the client, I don't think that should mean we store it publicly on Github where it's more visible and potentially scraped and spammed - I dunno what/who would, but seems like an unnecessary risk. Better to have it loaded from the .env file, it's already defined in there on the NestJS branch as `SENTRY_DSN` in `.env`.

**Review by tsa96 — CHANGES_REQUESTED** [review]
Looks like you've done all the actual Sentry stuff really well, just got to clean up the double commit and load the DSN from .env - we can discuss either in Discord anytime if you want.

I'm not going to merge this myself, I'd rather wait for Goc to have a look, which may not be until rewrite is done and we get Sentry on the backend up and running, though that's up to him. So rest assured if this takes a bit to get merged we've not forgotten about it and it'll get in round the time rewrite is done - in fact it's great to have this out of the way. Thanks for the first PR!!

File: client/package.json (hunk 1 above)
Comment (sergeevNick) [reply client/package.json]: Done

**Comment by tsa96** [comment]
Not planning on doing Sentry for frontend

**Pull request closed without merging**